README.md

```diff
@@ -3762,3 +4352,412 @@ return Http::withOptions([
+}
+
+if (request()->input('name')) {
+    // example.org/?name=0
+    // fails to apply query filter because evaluates to false
+    $query->where('name', request('name'));
+}
+
+if (request()->filled('key')) {
+    // correct way to check if get parameter has value
+}
+```
+
+Tip given by [@mc0de](https://twitter.com/mc0de/status/1465209203472146434)
+
+### Dust out your bloated route file
+Dust out your bloated route file and split it up to keep things organized
+```php
+class RouteServiceProvider extends ServiceProvider
+{
+    public function boot()
+    {
+        $this->routes(function () {
+            Route::prefix('api/v1')
+                ->middleware('api')
+                ->namespace($this->namespace)
+                ->group(base_path('routes/api.php'));
+                
+            Route::prefix('webhooks')
+                ->namespace($this->namespace)
+                ->group(base_path('routes/webhooks.php'));
+    
+            Route::middleware('web')
+                ->namespace($this->namespace)
+                ->group(base_path('routes/web.php'));
+                
+            if ($this->app->environment('local')) {
+                Route::middleware('web')
+                    ->namespace($this->namespace)
+                    ->group(base_path('routes/local.php'));
+            }
+        });
+    }
+}
+```
+
+Tip given by [@Philo01](https://twitter.com/Philo01/status/1466068376330153984)
+
+### You can send e-mails to a custom log file
+In Laravel you can send e-mails to a custom log file.
+
+You can set your environment variables like this:
+
+```
+MAIL_MAILER=log
+MAIL_LOG_CHANNEL=mail
+```
+
+And also configure your log channel:
+```php
+'mail' => [
+    'driver' => 'single',
+    'path' => storage_path('logs/mails.log'),
+    'level' => env('LOG_LEVEL', 'debug'),
+],
+```
+
+Now you have all your e-mails in /logs/mails.log<br>
+It's a good use case to quickly test your mails.
+
+Tip given by [@mmartin_joo](https://twitter.com/mmartin_joo/status/1466362508571131904)
+
+### Markdown made easy
+Laravel provides an interface to convert markdown in HTML out of the box, without the need to install new composer packages.
+```php
+$html = Str::markdown('# Changelogfy')
+```
+
+Output:
+```
+<h1>Changelogfy</h1>
+```
+
+Tip given by [@paulocastellano](https://twitter.com/paulocastellano/status/1467478502400315394)
+
+### Simplify if on a request with whenFilled() helper
+We often write if statements to check if a value is present on a request or not.<br>
+You can simplify it with the `whenFilled()` helper.
+
+```php
+public function store(Request $request)
+{
+    $request->whenFilled('status', function (string $status)) {
+        // Do something amazing with the status here!
+    }, function () {
+        // This it called when status is not filled
+    });
+}
+```
+
+Tip given by [@mmartin_joo](https://twitter.com/mmartin_joo/status/1467886802711293959)
+
+### Use observers if you need to listen to many events
```

Comment:
@krekas I don't think this tip is worth adding. It's a very simple Observer common sense example.